### PR TITLE
Send ETH query param in lower case

### PR DESCRIPTION
**Pull request by pgarciaegido**

### CODE REVIEW

File: packages/core/base/src/services/crossmintModalService.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ type MintQueryParams = {
     clientVersion: string;
     mintConfig: string;
     whPassThroughArgs?: string;
-    paymentMethod?: paymentMethods;
+    paymentMethod?: string;
```
Comment (angel-paella) [inline packages/core/base/src/services/crossmintModalService.ts:17]: why this change?

Comment (pgarciaegido) [reply packages/core/base/src/services/crossmintModalService.ts:17]: Becuase the type is `"fiat" | "ETH"` and we are technically passing `eth`
